packages/agent/src/invitation/handler.ts

```diff
@@ -35,44 +34,51 @@ import {
   W3cCredentialRepository,
   getJwkFromKey,
   parseMessageType,
-} from '@credo-ts/core'
-import { supportsIncomingMessageType } from '@credo-ts/core/build/utils/messageType'
-import { OpenId4VciCredentialFormatProfile } from '@credo-ts/openid4vc'
-import { getHostNameFromUrl } from '@package/utils'
-import { filter, first, firstValueFrom, merge, timeout } from 'rxjs'
+} from "@credo-ts/core";
+import { supportsIncomingMessageType } from "@credo-ts/core/build/utils/messageType";
+import { OpenId4VciCredentialFormatProfile } from "@credo-ts/openid4vc";
+import { getHostNameFromUrl } from "@package/utils";
+import { filter, first, firstValueFrom, merge, timeout } from "rxjs";
 
-import { extractOpenId4VcCredentialMetadata, setOpenId4VcCredentialMetadata } from '../openid4vc/metadata'
+import {
+  extractOpenId4VcCredentialMetadata,
+  setOpenId4VcCredentialMetadata,
+} from "../openid4vc/metadata";
 
 export const receiveCredentialFromOpenId4VciOffer = async ({
   agent,
   data,
   uri,
 }: {
-  agent: FullAppAgent
+  agent: FullAppAgent;
   // Either data itself (the offer) or uri can be passed
-  data?: string
-  uri?: string
+  data?: string;
+  uri?: string;
 }) => {
-  let offerUri = uri
+  let offerUri = uri;
 
   if (!offerUri && data) {
     // FIXME: Credo only support credential offer string, but we already parsed it before. So we construct an offer here
     // but in the future we need to support the parsed offer in Credo directly
-    offerUri = `openid-credential-offer://credential_offer=${encodeURIComponent(JSON.stringify(data))}`
+    offerUri = `openid-credential-offer://credential_offer=${encodeURIComponent(
+      JSON.stringify(data)
+    )}`;
   } else if (!offerUri) {
-    throw new Error('either data or uri must be provided')
+    throw new Error("either data or uri must be provided");
```

Comment:
why the formatting change?

Reply:
I still have prettier setup, will run biome again...just a mistake.